descriptors/ModuleDescriptor-template.json

```diff
@@ -488,13 +488,13 @@
       ]
     },
     {
-      "id": "instance-bulk",
-      "version": "0.1",
+      "id": "record-bulk",
```

Comment:
Given that this interface is being renamed, please could it be given a name that provides more context than only the term `record`.

What records are involved? If it is any inventory record, could `inventory-record-bulk` be an acceptable name?

This name takes up a very general space and could easily conflict with other interface names that are not related to inventory at all.

Reply:
make sense, updated, thanks!